Week3/HW3/3_moreJavascript/3_objFunction.js

```diff
@@ -0,0 +1,15 @@
+let animal = {type: "dog", name: "Pino", color: "Brown", age: "14"}
+
+function getObjectValue(obj) {
+    let txt = '';
+    for (param in obj) { 
+        txt += obj[param] + " "
+    }; 
+    return txt;
+}
```

Comment:
👍 for the function name. However, you missed providing `param` a keyword while creating it in line 5. 😱

Reply:
I added, thank you for your attention.